<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupSettings</class>
 <widget class="QMainWindow" name="groupSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>561</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 126, 126);
border-color: rgb(0, 0, 0);
border-radius:16px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="deleteGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>400</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgb(255, 0, 0);
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Delete Group</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="companyLogo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>15</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 4);
color: rgb(0, 0, 0);
font: 75 italic 26pt &quot;Century Gothic&quot;;
</string>
     </property>
     <property name="text">
      <string>Groups Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeNameBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>400</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: green;
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Change Name</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="groupList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>211</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
border-radius: 0px;
color:rgb(255, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);
font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Group List</string>
     </property>
    </widget>
    <widget class="QListWidget" name="groupUsersList">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>211</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
border-radius: 0px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 4);
font: 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Users in Group</string>
     </property>
    </widget>
    <widget class="QPushButton" name="demoteBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>500</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgb(255, 0, 0);
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Demote User</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="promoteBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: green;
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Promote User</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="removeUserBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>450</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: rgb(255, 0, 0);
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Remove User</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="addUserBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>450</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
background-color: green;
border-radius:16px;
}
QPushButton:hover
{
   background-color:white;
}</string>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
